### PR TITLE
Add new permalink param: `post_title`

### PR DESCRIPTION
**Pull request by nt3rp**
To avoid breaking any existing setups, create a new param which represents the title of a post.

Also, addresses #881 (as far as the problem I described there)

### CODE REVIEW

File: lib/plugins/filter/post_permalink.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ function postPermalinkFilter(data){
     id: data.id || data._id,
     title: data.slug,
```
Comment (nt3rp) [inline lib/plugins/filter/post_permalink.js]: I think that the proper fix would be to change this line, to this:

    title: util.slugize(data.title, {transform: 1}),

... but that could break existing setups :disappointed:

**Comment by tommy351** [comment]
What's different between slug and post_title?

**Comment by nt3rp** [comment]
From my understanding, `slug` in the permalink is equivalent to `new_post_name` minus the file extension. `post_title` is the slugized version of the actual post title.

The problem mostly arises in cases like #881.

**Comment by tommy351** [comment]
I see.

**Pull request merged**